src/runtime/ScriptClassConstructorFunctionObject.h

```diff
@@ -82,8 +90,8 @@ class ScriptClassConstructorFunctionObject : public ScriptFunctionObject {
     Object* m_homeObject;
     // We needs to store class source code for toString(). because class constructor stores its source code
     String* m_classSourceCode;
-    TightVector<std::pair<EncodedValue, EncodedValue>, GCUtil::gc_malloc_allocator<std::pair<EncodedValue, EncodedValue>>> m_instanceFieldInitData;
-    VectorWithNoSize<EncodedValue, GCUtil::gc_malloc_allocator<EncodedValue>> m_staticFieldInitData;
+    TightVector<std::tuple<EncodedValue, EncodedValue, size_t>, GCUtil::gc_malloc_allocator<std::tuple<EncodedValue, EncodedValue, size_t>>> m_instanceFieldInitData;
+    VectorWithNoSize<std::tuple<EncodedValue, size_t>, GCUtil::gc_malloc_allocator<std::tuple<EncodedValue, size_t>>> m_staticFieldInitData;
```

Comment:
It seems that `size_t` member in std::tuple represents `ClassPrivateFieldKind` enum value.
IMHO it's better to clearly define it as `ClassPrivateFieldKind`, so that we can easily recognize each member.
Besides, we could reduce the total size of vectors too.

Reply:
This is intended type
bdwgc needs to aligned address for find pointers. so I turned type into size_t